# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: layout/generic/crashtests/1553824.html
# Commit: f12c5ebd0bfc
# Full Hash: f12c5ebd0bfce0915e7979bc95e7e8781c2f3a4f
# Author: Mats Palmgren <mats@mozilla.com>
# Date: 2019-05-24 21:49:59
# Description:
#   Bug 1553824 - [css-grid-2] Invalidate any stored used track sizes on subgrids too.  r=dholbert
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D32372
# ==============================================================================

diff -r 386097a10f84 -r f12c5ebd0bfc layout/generic/crashtests/1553824.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/layout/generic/crashtests/1553824.html	Thu May 23 21:11:48 2019 +0200
@@ -0,0 +1,86 @@
+<body onload="test()">
+<script>
+function test() {
+  document.body.offsetWidth;
+  let grid = document.querySelector(".container");
+  grid.style = "grid-template-columns:none";
+}
+</script>
+
+<style>
+body {
+  font: 1.2em Arial, Verdana, sans-serif;
+  background-color: #fff;
+}
+
+.container {
+  display: grid;
+  gap: 10px;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-template-rows: auto auto auto;
+}
+
+header,
+footer,
+aside,
+li,.box {
+  background-color: rgb(120, 70, 123);
+  border: 5px solid rgb(88, 55, 112);
+  color: #fff;
+  border-radius: 5px;
+  padding: 20px;
+}
+
+aside {
+  grid-row: 1 / -1;
+  grid-column: 1;
+}
+
+header {
+  grid-column: 2 / -2;
+}
+
+ul {
+  gap: 10px;
+  grid-row: 2;
+  grid-column: 2 / -1;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  display: grid;
+  grid-template-columns: subgrid;
+}
+
+footer {
+  grid-row: 3;
+  grid-column: 2 / -1;
+}
+
+.box {
+  grid-column: -2;
+  grid-row:1;
+}
+
+</style>
+
+
+<div class="container">
+  <header>This is my header</header>
+  <div class="box"></div>
+  <aside>I should stretch from the top to the bottom of the grid</aside>
+  <ul>
+    <li>A</li>
+    <li>B</li>
+    <li>C</li>
+    <li>D</li>
+    <li>E</li>
+    <li>F</li>
+    <li>G</li>
+    <li>H</li>
+    <li>I</li>
+    <li>J</li>
+    <li>K</li>
+  </ul>
+  <footer>I am a footer</footer>
+</div>
+